subsys/bluetooth/crypto/bt_crypto.c

```diff
@@ -7,12 +7,12 @@
 
```

Comment:
Do we need the `bt_` prefix on the file or can we rename the file as well?

Reply:
I guess if we would rename bt_crypto.h to crypto.h, should bt_str.h be renamed as well (in a different PR)?

Reply:
It's a good point. We have `subsys/bluetooth/host/crypto.c` as well, but that is not a problem. Duplicating filenames in general is not a problem. I assume we will migrate the rest of `host/crypto.c` to `crypto/bt_crypto.c` at some point. I am OK with both. @theob-pro @alwa-nordic @jori-nordic @jhedberg opinions?